environment/src/main/java/com/sequenceiq/environment/network/dao/domain/BaseNetwork.java

```diff
@@ -98,6 +98,8 @@
     @Column(nullable = false)
     private String resourceCrn;
 
+    private String privateDnsZoneId;
```

Comment:
This and the privateEndpointsEnabled come from the base network however they Azure specific parameters and should be part of the [BaseNetwork's AzureParams object](https://github.com/hortonworks/cloudbreak/blob/2.54.0-b37/structuredevent-model/src/main/java/com/sequenceiq/environment/network/dto/NetworkDto.java#L32)